docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -517,7 +533,7 @@ The `dayPickerClasses` variable has been renamed `dayCalendarClasses` to be cons
 
 The `dateLibInstance` prop of `LocalizationProvider` does not work with `AdapterDayjs` anymore.
 This prop was used to set the pickers in UTC mode before the implementation of a proper timezone support in the components.
-You can learn more about the new approach on the [dedicated doc page](https://mui.com/x/react-date-pickers/timezone/).
+Learn more about the new approach on the [dedicated doc page](https://mui.com/x/react-date-pickers/timezone/).
```

Comment:
```suggestion
Learn more about the new approach on the [dedicated doc page](/x/react-date-pickers/timezone/).
```
Here Grammarly suggests the previous usage (You can learn) though. 🤔

Reply:
We had 2 "Look" and 1 "You can look" so I took the most common
But if Grammarly prefers "You can look" I will use that one :ok_hand: